src/wirecloud/platform/static/js/wirecloud/ui/WiringEditor/ComponentGroup.js

```diff
@@ -129,4 +131,21 @@
         this.imageElement.parentElement.appendChild(document.createTextNode(utils.gettext("No image available")));
     };
 
+    var orderVersions = function orderVersions(versions) {
+        versions = versions.map(function (value) {
+            return new Wirecloud.Version(value);
+        }).sort(function (version1, version2) {
+            return version1.compareTo(version2);
+        }).map(function (version) {
+            return {
+                label: 'v' + version.text,
```

Comment:
Please, use `version.toString()` for the label instead of `version.text` as the later is not meant to be used for displaying versions to the end-users.